src/adapters/acala.ts

```diff
@@ -65,24 +65,25 @@ export const karuraRoutersConfig: Omit<CrossChainRouterConfigs, "from">[] = [
       weightLimit: ACALA_DEST_WEIGHT,
     },
   },
-  //   {
-  //     to: "kintsugi",
-  //     token: "LKSM",
-  //     xcm: {
-  //       fee: { token: "LKSM", amount: "186480000" },
-  //       weightLimit: ACALA_DEST_WEIGHT,
-  //     },
-  //   },
+  {
+    to: "kintsugi",
+    token: "LKSM",
+    xcm: {
+      // fees in chopsticks tests: 186_480_000
+      fee: { token: "LKSM", amount: "200000000" },
```

Comment:
That's cutting it a bit close. This is expected to vary whenever we do a polkadot version bump. Let's add a zero since it's super cheap anyway

Reply:
Too easy, done in commit [69748a1](https://github.com/interlay/bridge/pull/51/commits/69748a166ebfb7b9e5518b94af02f17dca7f6986).